chore: update project name and logo (#125)
coreruleset/documentation: content/concepts/_index.md

---

# How CRS Works

> Deep dive in core concepts of the CRS in this chapter.

Author: ```suggestion
Deep dive into core CRS concepts in this chapter.
```
